.github/workflows/release-publish.yml

```
@@ -132,23 +130,4 @@ jobs:
             ${{ secrets.DOCKERHUB_USERNAME }}/fosslight:latest
             ${{ secrets.DOCKERHUB_USERNAME }}/fosslight:${{ github.event.release.tag_name }}
           cache-from: type=gha
-          cache-to: type=gha,mode=max
-
-  notify:
-    if: always()
-    needs:
-      - update-changelog
-      - build
-      - deploy-demo
-      - deploy-image
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Slack Workflow Notification
-        uses: Gamesight/slack-workflow-status@master
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
-          channel: '#release'
-          name: 'FOSSLight Bot'
-          icon_url: https://github.com/fosslight.png
+          cache-to: type=gha,mode=max
```

In the POSIX standard, a line is defined as: `A sequence of zero or more non- <newline> characters plus a terminating <newline> character`. We should follow this standard and add a newline at the end of a file.

```suggestion
          cache-to: type=gha,mode=max
```

---

@che-so 님, 종우님 커맨트 확인 후 수정 완료되시면 Resolve conversation해주십시오.